src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss

```diff
@@ -69,6 +69,34 @@
     }
     /* stylelint-enable */
   }
+
+  .ecl-link--cta {
```

Comment:
the button uses class `ecl-button--call`. We should keep some consistency (`ecl-link--call`)

Reply:
I disagree :) 
1. a link is not a button, there's no reason to maintain consistency between them
2. `cta` is much clearer than `call`

Reply:
ok, as agreed, we will keep it with `cta` here and update the buttons to use the same name